report-service/src/main/resources/web/template/report.hbs

```diff
@@ -0,0 +1,146 @@
+<div class="overops-quality-report">
+    <div class="d-flex align-center mt-3">
+        <div class="mr-2">
+            {{svg "web/img/overops-logo.svg"}}
+        </div>
+        <h1>Quality Report</h1>
+    </div>
+    <div class="alert {{data.statusCode.style}} d-flex align-center">
+        {{svg data.statusCode.svg}}
+        {{data.statusMsg}}
+    </div>
+    <h3>Report Summary</h3>
+    <table id="reportSummary" class="table table-striped w-25 mb-5">
+        <thead>
+            <tr>
+                <th>Gate</th>
+                <th>Status</th>
+                <th>Errors</th>
+            </tr>
+        </thead>
+        <tbody>
+            {{#with data.newErrorsTestResults}}
+                {{> reportSummary data.newErrorsTestResults}}
+            {{/with}}
+            {{#with data.resurfacedErrorsTestResults}}
+                {{> reportSummary data.resurfacedErrorsTestResults}}
+            {{/with}}
+            {{#with data.totalErrorsTestResults}}
+                {{> reportSummary data.totalErrorsTestResults}}
+            {{/with}}
+            {{#with data.uniqueErrorsTestResults}}
+                {{> reportSummary data.uniqueErrorsTestResults}}
+            {{/with}}
+            {{#with data.criticalErrorsTestResults}}
+                {{> reportSummary data.criticalErrorsTestResults}}
+            {{/with}}
+            {{#with data.regressionErrorsTestResults}}
+                {{> reportSummary data.regressionErrorsTestResults}}
+            {{/with}}
```

Comment:
I think we decided the regression table can be removed.

Reply:
keep it based on whether this changes in the future. Can be removed by not adding the query parameters for it.